pipelines/install_pipeline.yml

```diff
@@ -29,23 +29,27 @@
     - role: katello
```

Comment:
This is probably also incorrect and might pull in `katello` as a `foreman_installer_additional_packages` on the proxy.

Additionally we should probably skip the client repos on Foreman as well since those are supposed to be optional but that's for another PR.